test/unit/utils/caseTabTypeHelper.js

```diff
@@ -65,7 +65,15 @@ const nonProdTabIds = [
   'marriageCertificate',
   'coRespondent',
   // 'SolicitorCoRespondent', - TODO - uncomment this when we go live with AOS pack 2
-  'serviceApplication',
+  'serviceApplicationForPetSol',
+  'serviceApplicationForCitizen',
+  'serviceApplicationForCourtAdmin',
+  'serviceApplicationCourtAdminBeta',
```

Comment:
This is the only variable without a `for` in `serviceApplicationFor`.
It may be something you want to update

Reply:
That's because I didn't want to exceed 32 characters (which I believe it CCD's limit).